ts/pages/account/account_detail.tsx

```diff
@@ -15,17 +16,30 @@ export const AccountDetail: React.StatelessComponent<AccountDetailProps> = ({
     userEthAddress,
     userImageSrc,
 }) => {
-    const truncatedAddress = utils.getAddressBeginAndEnd(userEthAddress, 7, 3);
+    const truncatedAddress = utils.getAddressBeginAndEnd(userEthAddress, 7, 5);
 
     return (
         <Wrap>
-            <Avatar>
-                {
-                    userImageSrc
-                    ? <img src={userImageSrc} />
-                    : <Jazzicon seed={generateUniqueId(userEthAddress)} diameter={60} isSquare={true} />
-                }
-            </Avatar>
+            <HorizontalDiv>
+                <Avatar>
+                    {
+                        userImageSrc
+                        ? <img src={userImageSrc} />
+                        : <Jazzicon seed={generateUniqueId(userEthAddress)} diameter={60} isSquare={true} />
+                    }
+                </Avatar>
+                <ButtonWrapper>
+                    <Button
+                        href="/zrx/account/activity"
+                        isWithArrow={true}
+                        isAccentColor={true}
+                        shouldUseAnchorTag={true}
+                        target="_blank"
+                    >
+                        Show all activity
+                    </Button>
```

Comment:
If you use `to` instead of `href` it will be a smooth page transition rather than a full reload. Also could you define the path in `WebsitePaths` in `ts/types` for consistency.
```suggestion
                    <Button
                        to={WebsitePaths.AccountActivity}
                        isWithArrow={true}
                        isAccentColor={true}
                        shouldUseAnchorTag={true}
                    >
                        Show all activity
                    </Button>
```

Reply:
will do!

Reply:
nice that works much better 👍